.github/workflows/static-test.yml

```diff
@@ -0,0 +1,23 @@
+name: static-test
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Run static test
+      uses: aabadie/riot-action@v1
+      env:
+        CI_BASE_BRANCH: master
+      with:
+        cmd: make static-test
```

Comment:
Maybe just add a `git fetch origin master:master` before here?

Reply:
I pushed a commit to https://github.com/aabadie/riot-action to do that in the entrypoint.sh.

Reply:
I think you messed up capitalization [here](https://github.com/aabadie/riot-action/blob/master/entrypoint.sh#L5).

Any reason you don't use `riot:riotdocker` as a base [here](https://github.com/aabadie/riot-action/blob/master/Dockerfile#L1)?

Lastly, are there plans to move that repo to the RIOT-OS once this is merged?

Reply:
> I think you messed up capitalization here.

It works the same in URLs

> Any reason you don't use riot:riotdocker as a base here?

Size, 4.7GB for riotdocker, 600MB for riot-actions. Only drawback is duplication in Dockerfiles. But anyway, I think we should rethink our Docker strategy and avoid to have everything (all toolchains) in the same place but that's out of the scope here.

> are there plans to move that repo to the RIOT-OS once this is merged?

That would be better indeed.

Reply:
> > I think you messed up capitalization here.
> 
> It works the same in URLs

So just for shits and giggles you made it lower-case ;-P?

Reply:
The tools-buildtest now seems to have a problem as well :-/ https://github.com/aabadie/RIOT/runs/840950043?check_suite_focus=true#step:4:5